config/tgl-h.toml

```diff
@@ -0,0 +1 @@
+tgl.toml
```

Comment:
Should there be a newline here. It's showing up in red bold here ??

Reply:
This is a soft-link and git could not show correctly here.